doc/UsersGuide/Reference/Glossary.rst

```diff
@@ -227,7 +227,8 @@ Glossary
       A central location in which files (e.g., data, code, documentation) are stored and managed. 
 
    RRFS
-      The `Rapid Refresh Forecast System <https://gsl.noaa.gov/focus-areas/unified_forecast_system/rrfs>`__ (RRFS) is NOAA's next-generation convection-allowing, rapidly-updated, ensemble-based data assimilation and forecasting system currently scheduled for operational implementation in 2024. It is designed to run forecasts on a 3-km :term:`CONUS` domain. 
+      The `Rapid Refresh Forecast System <https://gsl.noaa.gov/focus-areas/unified_forecast_system/rrfs>`__ (RRFS) is NOAA's next-generation convection-allowing, rapidly-updated, ensemble-based data assimilation and forecasting system currently scheduled for operational implementation in 2024. It is designed to run forecasts on a 3-km :term:`CONUS` domain, see also `NOAA Rapid Refresh Forecast System (RRFS) <https://registry.opendata.aws/noaa-rrfs/>`__. Experimental data for is currently available from the `AWS S3 NOAA-RRFS  <https://noaa-rrfs-pds.s3.amazonaws.com/index.html#rrfs_a/>`__ bucket starting 02/01/2024 for deteministic forecasts out to 60 hours at 00, 06, 12, and 18 UTC, and out to 18 hours for other cycles. Earlier dates, from 05/01/2023 to 01/31/2024, may contain only forecasts at 00, 06, 12, 18 UTC; user needs to verify that data exist for needed dates.
```

Comment:
```suggestion
      The `Rapid Refresh Forecast System <https://gsl.noaa.gov/focus-areas/unified_forecast_system/rrfs>`__ (RRFS) is NOAA's next-generation convection-allowing, rapidly-updated, ensemble-based data assimilation and forecasting system currently scheduled for operational implementation in 2024. It is designed to run forecasts on a 3-km :term:`CONUS` domain. Experimental data for RRFS is currently available from the `RRFS  data bucket (rrfs_a data)<https://registry.opendata.aws/noaa-rrfs/>`_ bucket starting 02/01/2024. It includes data for deterministic forecasts out to 60 hours at 00, 06, 12, and 18 UTC and out to 18 hours for other cycles. Earlier dates, from 05/01/2023 to 01/31/2024, may contain only forecasts at 00, 06, 12, 18 UTC; users need to verify that data exist for needed dates.
```
It seems like all the dates includes forecasts at 00, 06, 12, 18 UTC... I'm a little confused about the distinction here.

Reply:
I'm not sure if I understand @gspetro-NOAA, but to me, it reads like the longer forecasts are available 4x per day at the synoptic time, while all the other forecasts are shorter (1-5Z, 7-11Z, 13-17Z, 19-23Z). The same thing happens for RAP and HRRR. All 3 of these systems run a forecast at every hour of the day instead of just 4x per day like the global systems and HAFS.

Reply:
@christinaholtNOAA @gspetro-NOAA - note that I've added "... for other **hourly** cycles..." to clarify the forecast times available.